packages/library-base/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -67,6 +67,15 @@ interface MutableRealm : TypedRealm {
      */
     override fun <T : RealmObject> objects(clazz: KClass<T>): RealmResults<T>
 
+    /**
+     * TODO : query
+     */
+    override fun <T : RealmObject> query(
+        clazz: KClass<T>,
+        query: String,
```

Comment:
I think this needs a default value like `TRUEPREDICATE` otherwise you cannot run queries like `mutableRealm.query(Sample::class).find()`

Reply:
An overriding function is not allowed to specify default values for its parameters. The parent function is specifying `TRUEPREDICATE` though so that should be enough, I think.